frontend/src/metabase/nav/components/AppBar/AppBarLarge.tsx

```diff
@@ -35,26 +36,31 @@ const AppBarLarge = ({
   isNavBarVisible,
   isSearchVisible,
   isNewButtonVisible,
+  isBreadcrumbsVisible,
   isProfileLinkVisible,
   isCollectionPathVisible,
   isQuestionLineageVisible,
   onToggleNavbar,
   onLogout,
 }: AppBarLargeProps): JSX.Element => {
+  isNavBarOpen = isNavBarOpen && isNavBarVisible;
+
   return (
     <AppBarRoot isNavBarOpen={isNavBarOpen}>
       <AppBarLeftContainer isNavBarVisible={isNavBarVisible}>
-        <AppBarLogo
-          isNavBarOpen={isNavBarOpen}
-          isToggleVisible={isNavBarVisible}
-          onToggleClick={onToggleNavbar}
-        />
+        {isNavBarVisible && (
+          <AppBarLogo
+            isNavBarOpen={isNavBarOpen}
+            isToggleVisible={isNavBarVisible}
+            onToggleClick={onToggleNavbar}
+          />
+        )}
         <AppBarInfoContainer
           isVisible={!isNavBarOpen || isQuestionLineageVisible}
         >
           {isQuestionLineageVisible ? (
             <QuestionLineage />
-          ) : isCollectionPathVisible ? (
+          ) : isBreadcrumbsVisible && isCollectionPathVisible ? (
```

Comment:
WDYT about changing the selector of `isCollectionPathVisible` to take new settings into the account instead of doing here?

Reply:
I'll try that, didn't think of that at last night. But seeing this now, it's weird that there are now 2 booleans controlling this component.

Reply:
Done!